lib/android-tools-info.ts

```diff
@@ -320,4 +343,36 @@ export class AndroidToolsInfo implements NativeScriptDoctor.IAndroidToolsInfo {
 		const errors = this.validateAndroidHomeEnvVariable();
 		return !errors && !errors.length;
 	}
+
+	private getAndroidRuntimeVersionFromProjectDir(projectDir: string): string {
+		let runtimeVersion: string = null;
+		if (projectDir && this.fs.exists(projectDir)) {
+			const pathToPackageJson = path.join(projectDir, Constants.PACKAGE_JSON);
+
+			if (this.fs.exists(pathToPackageJson)) {
+				const content = this.fs.readJson<INativeScriptProjectPackageJson>(pathToPackageJson);
+				runtimeVersion = content && content.nativescript && content.nativescript["tns-android"] && content.nativescript["tns-android"].version;
```

Comment:
We can use constants here
`ANDROID_RUNTIME`, `NATIVESCRIPT_KEY`

Reply:
In case we use the constants, we have to modify the interface INativeScriptNode to has index signature. This will allow anyone to write `content.nativescript["tns-andorid"]` and the transpilation will succeed. The current approach does not allow incorrect key to be passed.